### DISCUSSION TITLE
Unlock issue #4367

### DISCUSSION THREAD

**Discussion by matthieusieben** [zed-industries/zed#20656] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

The issue https://github.com/zed-industries/zed/issues/4367 is locked, meaning we can no longer vote (👍) for it.

Because of this, the feature will only get lower and lower in the list of [priority projects](https://github.com/zed-industries/zed/issues/5393). It recently lost one place.

If people don't want to get notified, they can "unsubscribe" from the issue (since this was used as reason for locking that issue).

**Comment by SomeoneToIgnore** [comment]
Worry not, we're gradually working on this.

It's locked to silence all useless spam messages in it and e-mail notifications to dozens of people.
We would love to keep the voting unlocked, but it's a current GitHub limitation alas, so the choice has to be made between closed voting and spam messages drawbacks, we've picked the first one.

------------------

Speaking of spam, your initial submission is barely better: this belongs into Discussions, not Issues.
